src/web3/networkConfig.ts

```diff
@@ -246,6 +247,9 @@ const CAB_DELETE_ENCRYPTED_MNEMONIC_MAINNET = `${CLOUD_FUNCTIONS_MAINNET}/delete
 const SAVE_CONTACTS_ALFAJORES = `${CLOUD_FUNCTIONS_STAGING}/saveContacts`
 const SAVE_CONTACTS_MAINNET = `${CLOUD_FUNCTIONS_MAINNET}/saveContacts`
 
+const GET_POINTS_HISTORY_ALFAJORES = `${CLOUD_FUNCTIONS_STAGING}/getHistory`
```

Comment:
i just realised how generic these function names are :| this might be a pain for monitoring (e.g. going through GCP logs and wondering which history this getHistory endpoint is talking about)...perhaps we should have "points" in our function names....wdyt?

Reply:
i realise this is out of scope for this PR